README.md

```diff
@@ -49,7 +49,40 @@ export default App;
 ```
 The `data` array is an array of objects with { x, y } cordinates.
 
->Note: The graph width is resposive. So the it can be adjusted by paraent wrapper. Need to provide the height.
+>Note: The graph width is resposive. So that it can be adjusted by parent wrapper. Need to provide the height.
+
+React Scatter Graph for date inputs:
```

Comment:
// Add this lines:
Scatter graph is a useful tool for plotting date values. In order to do so, timestamps must be provided for the x-axis values.
// Add an image if possible